docs/index.rst

```diff
@@ -6,8 +6,220 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with files:
+
+- metadata (holds the list of errors code can trigger)
+- checks (holds the :code:`check_` functions)
+
+Both of them need to be *accessible to the zoo*.
```

Comment:
Mention that the metadata and checks filenames have to match. We we import the metadata files based on the python check filename.